client/views/resetpwd/resetpwd.js

```diff
@@ -0,0 +1,103 @@
+import $ from 'jquery';
+
+Accounts.onResetPasswordLink((token, done) => {
+  Router.go('resetpwd');
+});
+
+if (Accounts._resetPasswordToken) {
+  Session.set('resetPasswordVar', Accounts._resetPasswordToken);
+}
+
+function showChangePwdError(reason, parentElement, nextElement) {
+  if (typeof currentError !== 'undefined') {
+    Blaze.remove(currentError);
+  }
+  const parentNode = $('.' + parentElement)[0];
+  const nextNode = $('.' + nextElement)[0];
+  currentError = Blaze.renderWithData(Template.form_error, reason, parentNode, nextNode);
+}
+
+Template.resetpwd.onRendered(() => {
+  $('.formcontainer').hide().fadeIn(400);
+  $('#darker').hide().fadeIn(400);
+});
+
+Template.resetpwd.helpers({
+  resetPassword() {
+    return Session.get('resetPasswordVar');
+  },
+});
+
+Template.resetpwd.events({
+  'keypress #newpasswordconfirm': function (event, template) {
+    event.which = event.which || event.keyCode;
+    if (event.which === 13) {
```

Comment:
Set `13` to a named constant so we know which key it is without having to look it up.